### PR TITLE
RTEMIS-63: Created and added workflow in school_details mini_node.

### PR DESCRIPTION
**Pull request by jaswinsingh**
Ticket # : https://innoraft.atlassian.net/browse/RTEMIS-63

---
### What the PR does
`(Explain fixes, changes and impacts)`
- Created and added workflow in school_details mini_node.

### CODE REVIEW

File: config/install/user.role.school_admin.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+uuid: a04fd565-2e25-4b11-8956-c4f1564da6b5
```
Comment (kunal-sin) [inline config/install/user.role.school_admin.yml:1]: Pls remove this

File: config/install/user.role.authenticated.yml [hunk 2]
```diff
@@ -7,13 +7,15 @@ dependencies:
...
     - system
+    - workflow
 id: authenticated
 label: 'Authenticated user'
 weight: 1
 is_admin: false
 permissions:
   - 'access content'
   - 'access user profiles'
+  - 'create school_registration_verification workflow_transition'
```
Comment (kunal-sin) [inline config/install/user.role.authenticated.yml:18]: This is still needed? As we now have a new role.

File: config/install/user.role.school_admin.yml (hunk 1 above)
Comment (jaswinsingh) [reply config/install/user.role.school_admin.yml]: Updated

File: config/install/user.role.authenticated.yml (hunk 2 above)
Comment (jaswinsingh) [reply config/install/user.role.authenticated.yml:18]: Yes, this is added by the module, so that this role can participate in workflow.

**Pull request merged**